worldedit-core/src/main/java/com/sk89q/worldedit/extent/reorder/MultiStageReorder.java

```diff
@@ -201,11 +224,14 @@ public Operation resume(RunContext run) throws WorldEditException {
                 }
             }
 
-            stage1.clear();
-            stage2.clear();
-            stage3.clear();
+            if (blocks.isEmpty()) {
```

Comment:
Is this in preparation for multiple-tick operations? I don't see how this would ever be false, there's no break for the outer `while` loop above.

Reply:
Yeah I don't know what this was for